pages/common/git-credential.md

```diff
@@ -0,0 +1,16 @@
+# git credential
+
+> Retrieve and store user credentials.
+> More information: <https://git-scm.com/docs/git-credential>.
+
+- Display credential information, retrieving the username/password from config files:
```

Comment:
Why not?
```suggestion
- Display credential information, retrieving the username and password from configuration files:
```

Reply:
It looks great! It's not actually my own wording, it's from [here](https://git-scm.com/docs/git-credential).